asciidoc/courses/cypher-intermediate-queries/modules/5-pipelining-queries/lessons/04-c3-map-projection/lesson.adoc

```diff
@@ -2,35 +2,38 @@
 :type: challenge
 :order: 4
 :sandbox: true
-:updated-at: 2022-05-03T14:00:00Z
 
-== Using WITH for map projection to limit results
+== Using WITH for map projection
 
-Here is a query, a version of which  you saw in the previous lesson:
+This query uses `WITH` to return a map projection of the top 10 movies by IMDb rating.
 
-[source,cypher,role=partial]
+[source,cypher]
 ----
-MATCH (n:Movie)
-WHERE n.imdbRating IS NOT NULL and n.poster IS NOT NULL
-// Add WITH clause to return custom data for each movie
-ORDER BY n.imdbRating DESC LIMIT 4
-RETURN collect(n)
+include::sandbox.cypher[]
 ----
 
-Modify this query by adding a WITH clause that customizes the data returned for each Movie node to include:
+Modify this query to add the following properties to the map projection:
 
-* title
-* imdbRating
-* List of actor names
-* List of Genre names
+- `plot`
+- `released`
+- `countries`
 
-Then answer this  question:
+[%collapsible]
+.Click to reveal the solution
+====
+[source,cypher]
+----
+include::solution.cypher[]
+----
+====
+
+Click Complete when you are ready to move on.
 
-include::./questions/q3.adoc[leveloffset=+1]
+read::Complete[]
 
 [.summary]
 == Summary
 
-In this challenge, you wrote a query to use a `WITH` clause to return a subset of data using map projection.
+In this challenge, you used WITH to create a map projection.
```

Comment:
In this challenge, you used the `WITH` clause to create a map projection.

I think we should be intentional about wrapping Cypher syntax in backticks.